### PR TITLE
Grunt && SASS

### PR DESCRIPTION
**Pull request by Pwarde**
@raymonschouwenaar pls review.

### CODE REVIEW

File: firstWireFrame/dev/css/style.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
+/* line 7, ../sass/style.scss */
```
Comment (raymonschouwenaar) [inline firstWireFrame/dev/css/style.css]: You should put this file in the `.gitignore`

File: firstWireFrame/dev/sass/style.scss [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+    display: flex;
+    flex-direction: row;
+    &__list{
+        margin: 10px;
+        padding: 10px;
+        list-style: none;
+        font-weight: 400;
+    }
+    &__list-link {
```
Comment (raymonschouwenaar) [inline firstWireFrame/dev/sass/style.scss]: Make this `&__link` that is clear enough

File: firstWireFrame/dev/sass/style.scss [hunk 3]
```diff
@@ -0,0 +1,195 @@
+$text-color: #14080E;
```
Comment (raymonschouwenaar) [inline firstWireFrame/dev/sass/style.scss]: Looks good, but I suggest you use this structure [https://github.com/Frontendhero/frontend-css-sass-structure](https://github.com/Frontendhero/frontend-css-sass-structure)

File: firstWireFrame/readme.md [hunk 4]
```diff
@@ -11,15 +11,15 @@ After HTML, create the CSS section by section. Extra classes and IDs added where
...
-- CSS design patterns best practices (what does Competa teach?)
-  - [SMACSS](https://smacss.com/)
-  - [BEM](https://en.bem.info/)
-  - [KSS](https://github.com/kneath/kss)
-  - [OOCSS](https://github.com/stubbornella/oocss/wiki)
+- Using BEM, is ID in Input fields acceptable?
+
+Useful links for this assignment:
+- Medium article: [getting started with grunt](https://medium.com/@verpixelt/get-started-with-grunt-76d29dc25b01)
+- Competa CodePamoja [grunt slides](https://slides.com/codepamoja/grunt)
```
Comment (raymonschouwenaar) [inline firstWireFrame/readme.md]: Write down how people could use this en what they should have installed.

File: gruntfile.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
+module.exports = function(grunt) {
+
+    // Project configuration.
+    grunt.initConfig({
+
+        // imports the config data from the package.json
+        pkg: grunt.file.readJSON('package.json'),
+    
+        // connect: {
```
Comment (raymonschouwenaar) [inline gruntfile.js]: Remove

**Comment by Pwarde** [comment]
Updated in Pull request #4

**Pull request closed without merging**